### PR TITLE
Centered information for mobile view and moved info to right

### PR DESCRIPTION
**Pull request by AmanGotchu**
#29

### CODE REVIEW

**Comment by gracekfan** [comment]
the january 26-27 line isn't centered in mobile, but everything else seems good

**Comment by AmanGotchu** [comment]
Is that line wrapping to the next line? If the whole line fits then it should be centered. Is it never centered for you?

File: src/App.js [hunk 1]
```diff
@@ -59,27 +60,28 @@ class App extends React.Component{
...
     const isWebView = this.state.width > mobileThreshold;
     const stylesObj = isWebView ? styles.webView : styles.mobileView;
-    const { cloud2, cloud4 } = stylesObj;
+    const { cloud2, cloud4, infoAlignment } = stylesObj;
 
     return(
       <Background>
         <Flex
-          css={{width: "100vw", height: "100vh", overflow: "hidden", zIndex: "2" }}
+          css={{width: "100vw", height: "100vh", overflow: "hidden", zIndex: "2", justifyContent: infoAlignment}}
```
Comment (jay-khatri) [inline src/App.js]: justifyContent is already a flex property, so you can just add it as a prop instead.

**Comment by gracekfan** [comment]
it's only centered for iphone 6/7/8 plus, but everything else wraps to the next line

File: src/App.js [hunk 2]
```diff
@@ -30,7 +30,8 @@ const styles = {
     },
     cloud4: {
       height: "5vh"
-    }
+    },
+    infoAlignment: "center"
```
Comment (jay-khatri) [inline src/App.js]: I think you should add a default value for when its a web view. You can make the value `unset`, `inherit`, or `initial` depending on what you want.

Comment (jay-khatri) [reply src/App.js]: This is mainly because we don't wanna be passing around `undefined` objects.

File: src/App.js [hunk 3]
```diff
@@ -59,33 +65,35 @@ class App extends React.Component{
...
               />
             </Flex>
-            <Subtitle style={{ paddingBottom: "2vh" }}>January 26-27, 2019 • Texas A&M University</Subtitle>
-            <Subtitle>Registration opens October 24th, 2018</Subtitle>
-            <SubmitField fontSize="13px" width="200px"/>
-            <Subtitle style={{ paddingTop: "3vh" }}>
+            <Subtitle isWebView={isWebView} style={{ paddingBottom: "5vh" }}>January 26-27, 2019 • Texas A&M University</Subtitle>
+            <Subtitle isWebView={isWebView} >Registration opens October 24th, 2018</Subtitle>
+            <SubmitField fontSize="13px" width={inputWidth} />
+            <Subtitle isWebView={isWebView} style={{ paddingTop: "3vh"}}>
```
Comment (jay-khatri) [inline src/App.js]: `{ isWebView }`

File: src/App.js [hunk 4]
```diff
@@ -59,33 +65,35 @@ class App extends React.Component{
...
+                style={{ height: logoSize, marginBottom: "20px"}}
               />
             </Flex>
-            <Subtitle style={{ paddingBottom: "2vh" }}>January 26-27, 2019 • Texas A&M University</Subtitle>
-            <Subtitle>Registration opens October 24th, 2018</Subtitle>
-            <SubmitField fontSize="13px" width="200px"/>
-            <Subtitle style={{ paddingTop: "3vh" }}>
+            <Subtitle isWebView={isWebView} style={{ paddingBottom: "5vh" }}>January 26-27, 2019 • Texas A&M University</Subtitle>
+            <Subtitle isWebView={isWebView} >Registration opens October 24th, 2018</Subtitle>
+            <SubmitField fontSize="13px" width={inputWidth} />
```
Comment (jay-khatri) [inline src/App.js]: Rename this so that we can do `{ width }`

**Pull request merged**